docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -722,30 +722,11 @@ Details on Retryable Writes
 ---------------------------
 
 The driver implements two mechanisms for retrying writes: modern and legacy.
-The modern mechanism is used when the driver connects to a MongoDB 3.6+
-replica set or a sharded cluster, and the ``retry_writes`` client option is
-not set to false. This mechanism retries writes once on network errors
-(timeout, connection reset, etc.) as well as transient errors resulting
-from servers being unavailable due to initialization, shutdown, replica
-set elections and similar events (the corresponding errors are not master,
-node is recovering, etc.). Prior to retrying the write the driver attempts
-to select a new primary, since the server that was previously used is
-likely to no longer be usable.
-
-The legacy mechanism is used when the driver connects to MongoDB 3.4 or
-earlier clusters, or to MongoDB 3.6 or later standalone servers, or when
-``retry_writes`` client option is set to false. In this mode the driver will
-retry write operations once due to transient errors resulting from
-servers being unavailable due to initialization, shutdown, replica set
-elections and similar events, but will not retry writes on network errors
-like timeouts.
-
-Note that writes are always retried on transient server status-related
-errors like "not master" and "node is recovering", regardless of what
-``retry_writes`` option is set to.
-
-The following write methods used in day-to-day operations on collections will
-be automatically retried:
+As of driver version 2.9.0, the modern mechanism is used by default, and the
```

Comment:
Am I understanding correctly that on server versions below 3.6 will have no retryable writes (either modern or legacy) by default, and legacy retryable writes will only be enabled on those servers if the `retryWrites: false` option is specified? It's not super clear to me from the changes in the documentation whether this is the case or if legacy retryable writes are enabled by default for older servers still, so I think we should explicitly state whichever way it is.

Reply:
That is correct, what would you suggest as more clear language?

Reply:
I'd change the sentence to "As of driver version 2.9.0, the modern mechanism is used by default on servers that support it, and the legacy mechanism is deprecated and disabled by default on all server versions."

Reply:
Thank you, made this change.